.pre-commit-config.yaml

```diff
@@ -1,21 +1,29 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.2.0
-    hooks:
-    -   id: trailing-whitespace
-    -   id: end-of-file-fixer
-    -   id: check-added-large-files
-    -   id: check-merge-conflict
-    -   id: detect-private-key
-    -   id: check-toml
--   repo: https://github.com/doublify/pre-commit-rust
-    rev: v1.0
-    hooks:
-    -   id: cargo-check
-    -   id: clippy
-    -   id: fmt
+    - repo: https://github.com/pre-commit/pre-commit-hooks
+      rev: v3.2.0
+      hooks:
+          - id: trailing-whitespace
+          - id: end-of-file-fixer
+          - id: check-added-large-files
+          - id: check-merge-conflict
+          - id: detect-private-key
+          - id: check-toml
+    - repo: https://github.com/doublify/pre-commit-rust
+      rev: v1.0
+      hooks:
+          - id: cargo-check
+          - id: fmt
+    - repo: local
+      hooks:
+        - id: system
+          name: cargo clippy
+          description: Run the Clippy linter on the package.
+          entry: cargo clippy --all-features --tests
+          language: system
+          files: \.rs$
+          pass_filenames: false
```

Comment:
Any particular reason to disable `pass_filenames`? 

Leaving it `true` allows for per-changed-file clippy execution, usually with performance improvements as we shave off a bunch of execution time on unchanged files.

Reply:
i don't know to be honest. I copied this config from the hook that was already there - https://github.com/doublify/pre-commit-rust/blob/master/hooks.yaml.
you might well be right. want me to change it?